experiments/20161006-hackytaq-replicates/20161006-hackytaq-replicate.md

```diff
@@ -0,0 +1,95 @@
+## Goals
+
+Following up on Vivian's set of excellently-conducted prior experiments (date: [2016-09-20](../20160920-Taq-Mn-mutagenesis/20160912-mgcl2-taq-series.md)), I wanted to fill in the blanks w.r.t. the concentrations not yet tested and sequenced.
```

Comment:
Also, some concentrations were repeated to try to get a sense of the variation in the num of mutations compared to your experimental run. As we see, of course, the 0 mM control had an unusually high mutation rate, and so I think all of the data isn't good.